integration/spark2/src/main/scala/org/apache/carbondata/indexserver/DistributedPruneRDD.scala

```diff
@@ -142,7 +142,6 @@ private[indexserver] class DistributedPruneRDD(@transient private val ss: SparkS
         while (reader.nextKeyValue()) {
           blocklets.add(reader.getCurrentValue)
         }
-        reader.close()
```

Comment:
not close reader?

Reply:
nextKeyValue is already closing when next() returns false